DiscordBee.cs

```diff
@@ -1,13 +1,14 @@
-using DiscordRPC;
-using DiscordRPC.Logging;
-using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.Text;
-using System.Text.RegularExpressions;
-
 namespace MusicBeePlugin
 {
+  using System;
```

Comment:
Why did you move the usings inside the namespace?

Reply:
That's just something I accustomed myself to a long time ago because it's marginally (read: in rare cases) safer (see [this answer on SO](https://stackoverflow.com/a/151560)).